packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -716,3 +744,18 @@ bool defaultSchedulingStrategy({ int priority, SchedulerBinding scheduler }) {
     return priority >= Priority.animation.value;
   return true;
 }
+
+/// Performance information for a Flutter frame.
```

Comment:
we need much more information here, such as, how to get an instance of such a class, what information it stores, what modes it would be available in, the protocol by which it's sent over to IDEs or pointers to documentation on that protocol, etc.